online-training.md

```diff
@@ -13,20 +13,49 @@ training as well as specific tips on using [Zoom](https://zoom.us/).
   - Decide instructor and backup-instructor in case of network issues
   - Decide helpers. One helper should be responsible for monitoring
     Zoom, i.e. the chat window, hand-raising and other feedback
-
 - TODO: set up pre-lesson polling? (zoom feature) Maybe unnecessary in view of pre-workshop survey
 - Instructors and helpers should use a reliable camera and microphone.
   Computer microphone might not be enough since audio quality will depend on
   instructor's head angle and proximity to screen.
 
+
+
+### At the beginning of the session
+
+- Allow time at the beginning of the session to debug video/audio and to
+  arrange windows. This takes few minutes so better do not start with teaching
+  from minute 1.
+- We cannot assume that all Zoom participants have the same and up to date
+  client and some clients do not contain "sticky notes" feedback or a button to
+  raise hands so agree with participants on signals (e.g. typing `\hand` in the
+  chat window seems to be standard).-
+
+
+### Recording of sessions
+
```

Comment:
```suggestion

If you plan on recording and publishing the session, prepare in advance so that
you don't have a difficult editing job later.
Make sure that you (or users) don't show any personal or confidential information.
Think about what happens if users speak: do you ask for permission to publish
in advance (maybe encouraging people not to), or edit it out later (taking
your time later).

```